One-Electron-Property/IR-Intensities/intensities.py

```diff
@@ -0,0 +1,173 @@
+
+References:
+- Equations from [Yamaguchi:1993]
+- Conversion factors from NIST
+"""
+
+__authors__ = ["Kirk C. Pearce", "Dominic A. Sirianni"]
+__credits__ = ["Kirk C. Pearce" , "Dominic A. Sirianni", 
+               "Andy C. Simmonet", "Zachary  Glick",
+               "C. David Sherrill", "Daniel R. Nascimento"]
+
+__copyright__ = "(c) 2014-2018, The Psi4NumPy Developers"
+__license__ = "BSD-3-Clause"
+__date__ = "2020-11-11"
+
+import numpy as np
+import psi4
+from psi4 import *
+np.set_printoptions(precision=10, linewidth=200, suppress=True)
+psi4.core.set_output_file('output.dat', False)
+
+# Setup
+numpy_memory = 2 
+
+# Useful Constants
+psi_c = psi4.constants.get("Natural unit of velocity") * 100                                                    # speed of light in cm/s
+psi_na = psi4.constants.na                                                                                      # Avogdro's number
+psi_alpha = qcel.constants.get("fine-structure constant")                                                       # finite-structure constant
+
+# Unit Conversions
+#
+# Energy conversions
+hartree2joule = psi4.constants.get("Atomic unit of energy")                                                     # Eh -> J
+#
+# Distance conversions
+psi_bohr2m = psi4.constants.bohr2m                                                                              # bohr -> m
+sqbohr2sqmeter = psi_bohr2m ** 2                                                                                # bohr^2 -> m^2
+psi_bohr2angstroms = psi4.constants.bohr2angstroms                                                              # bohr -> Ang
+#
+# Mass conversions
+amu2kg = psi4.constants.get("Atomic mass constant")                                                             # amu -> kg
+psi_au2amu = psi4.constants.au2amu                                                                              # m_e -> amu
+#
+# Dipole moment type conversions
+psi_dipmom_au2debye = psi4.constants.dipmom_au2debye                                                            # e a0 -> D (C m)
+conv_dip_grad_au2DAamu = (psi_dipmom_au2debye / psi_bohr2angstroms) * (1 / np.sqrt(psi_au2amu))                 # (e a0 / a0)/(m_e^(1/2)) -> (D/A)/(amu^(1/2))
+conv_cgs = 6.46047502185 * (10**(-36))                                                                          # (e a0)^2 -> (esu * cm)^2
+#
+# IR frequency conversions
+omega2nu = 1./(psi_c*2*np.pi)                                                                                   # omega (s^-1)  -> nu (cm^-1)
+psi4_hartree2wavenumbers = psi4.constants.hartree2wavenumbers                                                   # Eh -> cm^-1
+conv_ir_au2DAamu = conv_dip_grad_au2DAamu ** 2                                                                  # (e a0 / a0)^2/m_e -> (D/A)^2/amu
+conv_freq_au2wavenumbers = np.sqrt((1 / psi_au2amu) * hartree2joule / (sqbohr2sqmeter * amu2kg)) * omega2nu     # (Eh / (bohr^2 m_e)) -> cm^-1
```

Comment:
Might be possible to have `psi4.constants.conversion_factor("Hartree", "wavenumbers")` handle these items.

Reply:
Gotcha. Wasn't aware of this simplified conversion syntax. Is there documentation of the extendability of this `conversion_factor()` function. Would love to use it wherever possible.

Reply:
`(Eh / (bohr**2 * m_e)", "wavenumber")` - I would hope we could do this, but seems to get tripped up. It falls apart a bit on these very tricky conversions.

Reply:
I'm finding a few more that it can't do, e.g. `psi_au2amu = psi4.constants.conversion_factor("au", "amu")`. Do you want me to try and utilize this notation where I can still?

Reply:
After some time, I was able to get all the conversion factors to use `psi4.constants.conversion_factor()` except for the final IR intensity conversion factor where we go from `(D/A)^2 / amu` -> `km/mol`